apps/docs/src/styles/adapter-bootstrap.scss

```
@@ -4,6 +4,8 @@
 
 @use 'ember-bs' as ember-bs;
 @use 'tokens' as *;
+// End users: `@use '@ng-forge/dynamic-forms-bootstrap/styles/defaults' as df;`
```

Stylelint (17.11.1)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)